src/savegame_json.cpp

```diff
@@ -2783,7 +2785,9 @@ void vehicle::deserialize( JsonIn &jsin )
     data.read( "om_id", om_id );
     data.read( "faceDir", fdir );
     data.read( "moveDir", mdir );
-    data.read( "turn_dir", turn_dir );
+    int turn_dir_int;
+    data.read( "turn_dir", turn_dir_int );
+    turn_dir = units::from_degrees( turn_dir_int );
```

Comment:
it would be nice to add a serialize/deserialize function for this like the current ones in units.cpp

Reply:
(De)serialization of angles actually already works, but for backwards compatibility we still need to support reading integer degrees here.  The best option might have been to add support for both, but I decided to punt on that for the time being.